jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PlatformCheckerTest.java

```diff
@@ -58,39 +58,41 @@ public void testCheckManifestPlatform_mismatch() {
     ContainerConfigurationTemplate containerConfigJson = new ContainerConfigurationTemplate();
     containerConfigJson.setArchitecture("actual arch");
     containerConfigJson.setOs("actual OS");
-
-    PlatformChecker.checkManifestPlatform(buildContext, containerConfigJson);
-
-    Mockito.verify(eventHandlers)
-        .dispatch(
-            LogEvent.warn(
-                "the configured platform (configured arch/configured OS) doesn't match the "
-                    + "platform (actual arch/actual OS) of the base image (scratch)"));
+    Exception ex =
+        assertThrows(
+            PlatformNotFoundInBaseImageException.class,
+            () -> PlatformChecker.checkManifestPlatform(buildContext, containerConfigJson));
+    assertEquals(
+        "the configured platform (configured arch/configured OS) doesn't match the "
+            + "platform (actual arch/actual OS) of the base image (scratch)",
+        ex.getMessage());
```

Comment:
Perhaps we can use Truth here to assert on the message? Same below.

`import static com.google.common.truth.Truth.assertThat`
and `assertThat().hasMessageThat().isEqualTo()`

Reply:
Done!